dictionaries/custom_dict.txt

```diff
@@ -0,0 +1,634 @@
+100atestfet
+10px
+14th
+150p
+15t01
+19t02
+1atestfet
+46d2f86a255ece3daf244e2ca11d5be0f16cb633
+89ab
+8a6d3b
+9a
+AA
+ABI
+API
+APIs
+APR
+ASI
+AVCTL
+AcceptChitChatDialogue
+Agentverse
+AiEngine
+Almanac
+AlmanacContract
+ApiResponses
+BTC
+Bain
+Binance
+BirthDate
+Blockchain
+Blockchains
+BookingResponse
+Bosdet
+BroadcastError
+BroadcastExampleRequest
+BroadcastExampleResponse
+CL
+CLI
+CMA
+CMD
+CSV
+CTRL
+CW20
+Callout
+Caroline
+ChitChatDialogue
+ChitChatDialogueMessage
+ChitChatDialogueMessage
+ClientSession
+CoinToss
+Coinbase
+ConcludeChitChatDialogue
+CosmPy
+CosmWasm
+Cosmostation
+DD
+DDOS
+DDoS
+DENOM
+DNS
+DataBuyingAgent
+Databricks
+Datetime
+DeFi
+Debian
+Decentr
```

Comment:
what is this word? 😅

Reply:
Decentr?

It is in wallet docs

Reply:
Double checking this one

Reply:
It's a browser I have never heard of

Reply:
It is in pages/guides/fetch-network/fetch-wallet/web-wallet/get-started.mdx